report_qweb_pdf_watermark/models/report.py

```diff
@@ -19,8 +19,17 @@
     logger.error("ImportError: The PdfImagePlugin could not be imported")
 
 try:
-    from PyPDF2 import PdfFileReader, PdfFileWriter  # pylint: disable=W0404
-    from PyPDF2.utils import PdfReadError  # pylint: disable=W0404
+    try:
+        from PyPDF2 import PdfReader, PdfWriter  # pylint: disable=W0404
+        from PyPDF2.errors import PdfReadError  # pypdf >= 2.0
+    except ImportError:
+        from PyPDF2 import (  # pylint: disable=W0404
+            PdfFileReader as PdfReader,
+        )
+        from PyPDF2 import (
+            PdfFileWriter as PdfWriter,
+        )
+        from PyPDF2.utils import PdfReadError  # pypdf < 2.0
 except ImportError:
     logger.debug("Can not import PyPDF2")
```

Comment:
Would you consider making this a `warning`? With 2.12.1 we have everything until PdfReadError in which case setting the watermark goes fine until any exception occurs.

Reply:
happy to oblige, even though I don't think I understand what you wrote. Is there a version of the 2.x branch where the above fails? In 2.12.1 the import [should](https://github.com/py-pdf/pypdf/blob/2.12.1/PyPDF2/errors.py) work it seems to me.